tests/kuttl/test-runonnotready-cji/01-assert.yaml

```diff
@@ -0,0 +1,61 @@
+---
+apiVersion: cloud.redhat.com/v1alpha1
+kind: ClowdApp
+metadata:
+  name: puptoo
+  namespace: test-runonnotready-jobs
+status:
+  ready: false
```

Comment:
This assert is failing because your clowdapp is ready. You can slice this one of two ways. You can remove this assert and it should pass (or at least, not fail because of this specific problem) or you can try to force your clowdapp to not be ready, which if I'm understanding the feature correctly is probably what you want. Problem is I'm not sure how you'd force that. Maybe force the running imaghe to fail healthchecks?

Reply:
yeah, i made it a bad image instead so it fails to pull :)